app/views/content/shared/quotes/_quote_maths_gloria.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="quote-gloria">
```

Comment:
you don't need to add `<div class>` to quotes any more - @sarahcrack can advise!